gojira.sh

```diff
@@ -652,6 +660,23 @@ function set_snapshot_image_name {
 }
 
 
+function cli_envs {
+  if [[ ! -z $GOJIRA_CLI_ENVS ]]; then
```

Comment:
This conditional check is already on the function calling this one, right?

Reply:
it is and thinking more about it we should probably move the second check that occurs to be in the `add_egg` function since it falls over if there is nothing in the passed in egg. WDYT?